File: 1/1.hs
--multiplies x = filter d [0..x-1]
--  where d y = (mod y 5) == 0 || (mod y 3) == 0

multiplies x = filter d [0..x-1]
  where d y
          | mod y 3 == 0 = True
          | mod y 5 == 0 = True 
          | otherwise = False

main = print $ sum $ multiplies 1000

File: 1/1.cpp
#include <iostream>
#include <vector>

int main () {
  std::vector<int> multiplies;
  for (int i = 0; i < 1000; i++) {
    if (i % 5 == 0 or i % 3 == 0) {
      multiplies.push_back(i);
    }
  }
  int sum = 0;
  for (auto i : multiplies) {
    sum += i;
  }
  std::cout << sum << std::endl;
}